configure.ac

```diff
@@ -42,11 +42,18 @@ AC_SUBST(SHARED_VERSION_INFO)
 AM_GNU_GETTEXT([external])
 AM_GNU_GETTEXT_VERSION([0.19.7])
 AC_PROG_YACC
-AS_IF([test "$YACC" != 'bison -y'], [
-	YACC="\${top_srcdir}/build-aux/missing bison -y"
-	AC_MSG_NOTICE(no bison program found: only required for maintainers)
-	])
-AM_CONDITIONAL([HAVE_BISON], [test "$YACC" = 'bison -y'])
+  # Extract the first word of "$YACC", so it can be a program name with args.
+  set dummy $YACC; ac_word=$2
+  AC_MSG_CHECKING(whether $ac_word is GNU Bison)
```

Comment:
I would like you to not bother with `$ac_word` if all you want to do is invoke `$YACC --version`. This would make code a bit smaller. The message `checking whether bison -y is GNU Bison` would be still sufficient for indicating what configure is doing.

Reply:
The message `checking whether bison -y is GNU Bison` would be still sufficient for indicating what configure is doing.
Yes, it would, just looks a bit strange and is different from normal configure outputs. If requested I'll revert that part.

@westes your call: Do you want me to revert that part making configure.ac smaller (it doesn't count for the generated configure script which is much bigger) or stay with what the generated configure script normally does?